bootstrap/bacardi.sh

```diff
@@ -31,7 +31,11 @@ sync_node
 
 # NPM update
 if [ ! -f .last_update ] || [ package.json -nt .last_update ]; then
-  npm install && > .last_update
+  if is_windows_platform; then
+    npm install windows-build-tools --save-dev && npm install && > .last_update
```

Comment:
i don't like to call "npm install" twice.
actually i just added "windows-build-tools" in package.json.
but it made error in linux and mac.

and i searched to know how to distinguish os in install processing. and i found one. and i will try it in next patch.

Reply:
i changed it. it was worked well in any os.